features/array-from.yml.dist

```diff
@@ -0,0 +1,18 @@
+# Generated from: array-from.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+status:
+  baseline: high
+  baseline_low_date: 2016-09-20
+  baseline_high_date: 2019-03-20
+  support:
+    chrome: "45"
+    chrome_android: "45"
+    edge: "12"
+    firefox: "38"
```

Comment:
Note that this changed from 32 to 38, but I didn't check why because it's long ago and doesn't change the Baseline date.